src/main/java/core/basesyntax/service/impl/ProductDaoImpl.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.ProductDao;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ProductDaoImpl implements ProductDao {
```

Comment:
this class and ProductDao interface should be moved to the 'dao' and 'dao.impl' packages. Let's move them out of the 'service' package